src/util/agent_control.c

```diff
@@ -115,9 +114,6 @@ int main(int argc, char **argv)
             case 'L':
                 list_responses = 1;
                 break;
-            case 'e':
```

Comment:
Now that `e` is no longer an option, should we remove it from `"VehdlLcsjarmu:i:b:f:Rn" `?

Reply:
Fixed